### PR TITLE
RPC-DICT-ACCESS -- Enable ClusterRpcProxy dictionary access.

### PR DESCRIPTION
**Pull request by gianchub**
Enable dict like access for rpc proxy in the shell.

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -5,6 +5,15 @@ Here you can see the full list of changes between nameko versions. Versions
 are in form of *headline.major.minor* numbers. Backwards-compatible changes
 increment the minor version number only.
 
+Version 2.3.2
+-------------
+
+Released ...
```
Comment (gianchub) [inline CHANGES]: @matt

How does it work with this file?

Comment (mattbennett) [inline CHANGES]: Use `Released PENDING`

Comment (gianchub) [inline CHANGES]: Done, thank you.

**Comment by mattbennett** [comment]
Looks good, but you need to raise it against onefinestay:master. Do that and let davidszotten give some input

**Comment by gianchub** [comment]
Sure, thank you.

**Pull request merged**